Dockerfile

```diff
@@ -36,17 +36,11 @@ ENV JAVA_HOME=/opt/jdk-mini
 ENV PATH="$PATH:$JAVA_HOME/bin"
 ENV DIRPATH /pitchfork
 
-
 # Create some dirs and copy pitchfork jar
 RUN mkdir -p $DIRPATH
 COPY target/pitchfork.jar $DIRPATH/
 
 RUN chmod 755 $DIRPATH/pitchfork.jar
 WORKDIR $DIRPATH
 
-# Set timezone (for logs) and run pitchfork
-CMD export TZ=$(date +%Z) &&\
-    exec $JAVA_HOME/bin/java \
-         $JAVA_JVM_ARGS \
-         -jar \
-         pitchfork.jar
+CMD ["/bin/bash", "-c", "$JAVA_HOME/bin/java $JAVA_JVM_ARGS -jar pitchfork.jar"]
```

Comment:
will java be PID 1 with this? so that shutdown is graceful

Reply:
i'm going to revert this change.
this was for option 2 (force using bash) that still allows properties with dots.